mainwindow.cc

```diff
@@ -2207,85 +2204,73 @@ void MainWindow::editPreferences()
 
   if ( preferences.exec() == QDialog::Accepted )
   {
-    Config::Preferences p = preferences.getPreferences();
+    Config::Preferences newCfg = preferences.getPreferences();
 
     // These parameters are not set in dialog
-    p.zoomFactor = cfg.preferences.zoomFactor;
-    p.helpZoomFactor = cfg.preferences.helpZoomFactor;
-    p.wordsZoomLevel = cfg.preferences.wordsZoomLevel;
-    p.hideMenubar = cfg.preferences.hideMenubar;
-    p.searchInDock = cfg.preferences.searchInDock;
-    p.alwaysOnTop = cfg.preferences.alwaysOnTop;
-
-    p.proxyServer.systemProxyUser = cfg.preferences.proxyServer.systemProxyUser;
-    p.proxyServer.systemProxyPassword = cfg.preferences.proxyServer.systemProxyPassword;
-
-    p.fts.dialogGeometry = cfg.preferences.fts.dialogGeometry;
-    p.fts.matchCase = cfg.preferences.fts.matchCase;
-    p.fts.maxArticlesPerDictionary = cfg.preferences.fts.maxArticlesPerDictionary;
-    p.fts.maxDistanceBetweenWords = cfg.preferences.fts.maxDistanceBetweenWords;
-    p.fts.searchMode = cfg.preferences.fts.searchMode;
-    p.fts.useMaxArticlesPerDictionary = cfg.preferences.fts.useMaxArticlesPerDictionary;
-    p.fts.useMaxDistanceBetweenWords = cfg.preferences.fts.useMaxDistanceBetweenWords;
-    p.fts.ignoreWordsOrder = cfg.preferences.fts.ignoreWordsOrder;
-    p.fts.ignoreDiacritics = cfg.preferences.fts.ignoreDiacritics;
-
-    bool needReload = false;
+    newCfg.zoomFactor = cfg.preferences.zoomFactor;
+    newCfg.helpZoomFactor = cfg.preferences.helpZoomFactor;
+    newCfg.wordsZoomLevel = cfg.preferences.wordsZoomLevel;
+    newCfg.hideMenubar = cfg.preferences.hideMenubar;
+    newCfg.searchInDock = cfg.preferences.searchInDock;
+    newCfg.alwaysOnTop = cfg.preferences.alwaysOnTop;
+
+    newCfg.proxyServer.systemProxyUser = cfg.preferences.proxyServer.systemProxyUser;
+    newCfg.proxyServer.systemProxyPassword = cfg.preferences.proxyServer.systemProxyPassword;
+
+    newCfg.fts.dialogGeometry = cfg.preferences.fts.dialogGeometry;
+    newCfg.fts.matchCase = cfg.preferences.fts.matchCase;
+    newCfg.fts.maxArticlesPerDictionary = cfg.preferences.fts.maxArticlesPerDictionary;
+    newCfg.fts.maxDistanceBetweenWords = cfg.preferences.fts.maxDistanceBetweenWords;
+    newCfg.fts.searchMode = cfg.preferences.fts.searchMode;
+    newCfg.fts.useMaxArticlesPerDictionary = cfg.preferences.fts.useMaxArticlesPerDictionary;
+    newCfg.fts.useMaxDistanceBetweenWords = cfg.preferences.fts.useMaxDistanceBetweenWords;
+    newCfg.fts.ignoreWordsOrder = cfg.preferences.fts.ignoreWordsOrder;
+    newCfg.fts.ignoreDiacritics = cfg.preferences.fts.ignoreDiacritics;
```

Comment:
I think it would be better to keep the original `p` variable , this will make the changed line less.

Reply:
The rename change is in a separate commit. The changed line of each is not that much.